include/SDL_events.h

```diff
@@ -1151,6 +1151,40 @@ extern DECLSPEC Uint8 SDLCALL SDL_EventState(Uint32 type, int state);
  */
 extern DECLSPEC Uint32 SDLCALL SDL_RegisterEvents(int numevents);
 
+typedef void (*SDL_ModalLoopCallback)(void*);
```

Comment:
This should be (SDLCALL * SDL_ModalLoopCallback) like other public SDL func pointers